src/objects/culling.ts

```diff
@@ -17,25 +15,18 @@ export async function startCulling(): Promise<void> {
     toggle: 'renderable',
   })
 
-  callback = (): void => {
+  application.ticker.add(() => {
```

Comment:
Replaced all `Ticker.shared.add` calls with `application.ticker.add` because the shared ticker keeps ticking even after the application is torn down. The application ticker automatically stops when the application is destroyed.